app/views/admin/editions/_image_template.html.erb

```diff
@@ -67,7 +67,7 @@
             bold: true,
           },
           name: "readonly",
-          value: "!!#{index + 1}",
```

Comment:
What will happen to old documents that use the `!!n` syntax?

Reply:
@DilwoarH the old syntax will still be valid and working - please let me know if you noticed otherwise